pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -108,7 +108,26 @@ class TestReshaping(BaseDecimal, base.BaseReshapingTests):
 
 
 class TestGetitem(BaseDecimal, base.BaseGetitemTests):
-    pass
+
+    def test_take_basic(self):
```

Comment:
Move this to the generic base tests?

Reply:
Just added these for easier debugging. All of the are tested elsewhere.

I did notice a bug in the generic `test_take_non_na_fill_value`. I forgot to pass `allow_fill=True`, so we were doing NumPy style instead of pandas style. The tests happened to pass since `data_missing.take([-1, 1], fill_value)` happened to be the correct result.